browser/brave_profile_prefs.cc

```diff
@@ -180,10 +180,12 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
 
   // New Tab Page
   registry->RegisterBooleanPref(kNewTabPageShowBackgroundImage, true);
+  registry->RegisterBooleanPref(kNewTabPageShowBrandedBackgroundImage, true);
```

Comment:
this and `kBrandedWallpaperNotificationDismissed` should be moved into NewTabPageBrandedViewCounterFactory::RegisterProfilePrefs

Reply:
done